src/frontend/src/handler/create_source.rs

```diff
@@ -225,6 +225,21 @@ fn consume_string_from_options(
     ))))
 }
 
+fn get_json_schema_location(
+    row_options: &mut BTreeMap<String, String>,
```

Comment:
can we reuse the avro one?

Reply:
nope since `(None,None)` is valid for json schema.